CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ const { DynamicThreadPool } = require('poolifier')
 For cluster and thread pools, you can now only send and receive serializable `JSON` data.  
 _This is not a limitation by poolifier but NodeJS._
 
+#### Public properties renaming
+
+- Thread Pool's `numWorkers` is now `numberOfWorkers`
```

Comment:
Please we should indicate how old code looks like from an end user perspective and how  the new should look.
Please indicate that this change is into the constructor

Reply:
Seems you misunderstood one thing.
The one part was just a constructor parameter, its not a change for the user at all!
The other one is the Property that you can access like `myPool.numWorkers` and it's now `myPool.numberOfWorkers`
The user will find this directly without any problems

Also there were not any examples in the example folder that provides an example for exact using this property.
So it's just for advances users and they know what to do